config/initializers/pusher.rb

```diff
@@ -4,6 +4,6 @@
   Pusher.app_id = ENV['PUSHER_APP_ID']
   Pusher.key = ENV['PUSHER_APP_KEY']
   Pusher.secret = ENV['PUSHER_APP_SECRET']
-  Pusher.host = ENV['PUSHER_HOST'] if ENV['PUSHER_HOST']
+  Pusher.host = ENV['PUSHER_HOST'].dup if ENV['PUSHER_HOST']
   Pusher.port = ENV['PUSHER_PORT'].to_i if ENV['PUSHER_PORT']
```

Comment:
What do you think of use the safe navigator operator here? For example: `ENV["PUSHER_PORT"]$.to_i`

Reply:
I think this is a great idea, I will do this.